### PR TITLE
@broskoski => Annotate related artist on homepage with followed artist

### PR DESCRIPTION
**Pull request by mzikherman**
This is pretty ghetto.

Basically, the context object shouldn't be an `Artist`, but something custom, a struct of the artist and another object that represents the follow it is based on, or maybe we could get away with something as simple as a struct of `{href, follow_annotation}`. 

My JS-fu wasn't good enough to get that tho, so just went with this for now :(

```
{
  "data": {
    "home_page_module": {
      "title": "Works by Jasper Johns",
      "key": "related_artists",
      "context": {
        "href": "/artist/jasper-johns",
        "id": "jasper-johns",
        "follow_annotation": "Based on Andy Warhol"
      }
    }
  }
}
```

### CODE REVIEW

File: schema/artist/index.js [hunk 1]
```diff
@@ -44,6 +44,9 @@ const ArtistType = new GraphQLObjectType({
       id: {
         type: GraphQLString,
       },
+      follow_annotation: {
+        type: GraphQLString,
+      },
```
Comment (dzucconi) [inline schema/artist/index.js]: I really don't believe this should be here

Comment (dzucconi) [inline schema/artist/index.js]: To clarify: this is meaningless to any other call but the homepage artist. The schema can be enriched there.

Comment (mzikherman) [inline schema/artist/index.js]: I agree, but my JS-fu wasn't up to par! I don't know _how_ to do that...

**Comment by mzikherman** [comment]
Ok re-jiggered, 'tis better now!

It's just a struct of two artists.

**Comment by mzikherman** [comment]
This is ready to go Mr @broskoski

File: schema/home/context.js [hunk 2]
```diff
@@ -40,8 +41,16 @@ export const HomePageModuleContextFollowArtistsType = create(FollowArtists.type,
   isTypeOf: ({ context_type }) => context_type === 'FollowArtists',
 });
 
-export const HomePageModuleContextRelatedArtistsType = create(Artist.type, {
-  name: 'HomePageModuleContextRelatedArtists',
+export const HomePageModuleContextRelatedArtistType = new GraphQLObjectType({
+  name: 'HomePageModuleContextRelatedArtist',
+  fields: () => ({
+    artist: {
+      type: Artist.type,
```
Comment (broskoski) [inline schema/home/context.js]: nice

**Pull request merged**